doc/next-steps/filter-examples.md/$ne.md

```diff
@@ -0,0 +1,75 @@
+# $ne
+
+Records that do not exactly match the given value will be returned.
```

Comment:
assuming this is case sensitive but might be worth mentioning here as well. if `Workout` exists on name and we send `$ne=workout` it would still return because capital W